browser/net/brave_network_delegate_browsertest.cc

```diff
@@ -67,21 +82,28 @@ class BraveNetworkDelegateBrowserTest : public InProcessBrowserTest {
         browser()->profile(), brave_shields::ControlType::ALLOW, GURL());
   }
 
-  void AllowCookies() {
+  void AllowCookies(const EmbeddedTestServer* server) {
     brave_shields::SetCookieControlType(browser()->profile(),
                                         brave_shields::ControlType::ALLOW,
-                                        top_level_page_url_);
+                                        server->GetURL("a.com", "/"));
+  }
+
+  void BlockThirdPartyCookies(const EmbeddedTestServer* server) {
```

Comment:
I think this method name is confusing because it's not obvious in the tests that this is a site override and not the default. I think it's better to pass in the URL instead of the server

Reply:
or pass both